templates/layout.html

```diff
@@ -209,7 +185,19 @@
         </div>
       </div>
       <!-- Banner end -->
-      {{ template "mainNavigation" .MainMenuItems }}
+      {{ template "mainNavigation" . }}
+      {{ if not (or (eq .Active "index") (eq .Active "login") (eq .Active "register")) }}
+        <div id="mobile-search-bar" class="info-banner-center pb-2 pt-2 w-100 container">
```

Comment:
Sorry, for not noticing this earlier, but for widths in between 640 and 960 the header does not have the full width of the window.
![image](https://github.com/gobitfly/eth2-beaconchain-explorer/assets/109136188/0cb55826-c324-4bbc-8e5c-c2a6204f7e9c)

Reply:
is now full width